src/hm/miPayload.h

```diff
@@ -68,15 +71,96 @@ class MiPayload {
             if (mSerialDebug)
                 DPRINTLN(DBG_INFO, F("(#") + String(iv->id) + F(") Requesting Inv SN ") + String(iv->config->serial.u64, HEX));
 
-            uint8_t cmd = 0x09; //iv->getQueuedCmd();
-            DPRINTLN(DBG_INFO, F("(#") + String(iv->id) + F(") sendTimePacket"));
-            mSys->Radio.sendTimePacket(iv->radioId.u64, cmd, mPayload[iv->id].ts, iv->alarmMesIndex, false);
+            uint8_t cmd = iv->type == INV_TYPE_4CH ? 0x36 : 0x09; //iv->getQueuedCmd();
+            DPRINTLN(DBG_INFO, F("(#") + String(iv->id) + F(") prepareDevInformCmd"));
+            mSys->Radio.prepareDevInformCmd(iv->radioId.u64, cmd, mPayload[iv->id].ts, iv->alarmMesIndex, false, cmd);
             mPayload[iv->id].txCmd = cmd;
         }
 
         void add(Inverter<> *iv, packet_t *p) {
             DPRINTLN(DBG_INFO, F("MI got data [0]=") + String(p->packet[0], HEX));
 
+            if (p->packet[0] == (0x08 + ALL_FRAMES)) { // MI status response to 0x09
+                mPayload[iv->id].stsa = true;
+                /*decode here or memcopy payload for later decoding?
+                for decoding see
+                void MI600StsMsg (NRF24_packet_t *p){
+                  STAT = (int)((p->packet[11] << 8) + p->packet[12]);
+                  FCNT = (int)((p->packet[13] << 8) + p->packet[14]);
+                  FCODE = (int)((p->packet[15] << 8) + p->packet[16]);
+                #ifdef ESP8266
+                  VALUES[PV][5]=STAT;
+                  VALUES[PV][6]=FCNT;
+                  VALUES[PV][7]=FCODE;
+                #endif
+                }
+                */
+                DPRINTLN(DBG_INFO, F("Inverter ") + String(iv->id) + F(": status msg ") + p->packet[0]);
+            } else if (p->packet[0] == (0x12 + ALL_FRAMES)) { // MI status response to 0x11
```

Comment:
ist das richtig? das steht `0x12 + ...` muss das nicht 0x11 heißen?

Reply:
Das mit 0x92 paßt an der Stelle.
Die "kleinen MI" sind da etwas seltsam, weil zwei Antworten auf eine Anfrage kommen. Die "Großen" (4-kanaligen) sind dann wieder anders...

Reply:
Und ja, es kommen Antworten.
Allerdings (zumindest auf den ersten Blick) nicht mehr so "bilderbuchmäßig" wie in der vorherigen Version, vermutlich weil es noch an den repeats der requests auf allen Kanälen bei Ausbleiben der Antwort fehlt.